rust/processor/src/db/postgres/migrations/2024-06-27-213336_add_v1_migration_view/down.sql

```diff
@@ -0,0 +1,2 @@
+-- This file should undo anything in `up.sql`
+DROP VIEW IF EXISTS legacy_migration_v1.current_collection_datas
```

Comment:
Same with ;

Reply:
hmm i was still able to run the diesel. will add just in case